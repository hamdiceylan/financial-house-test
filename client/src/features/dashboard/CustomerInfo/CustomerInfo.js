import React from 'react'
import { Segment, Item, Header } from 'semantic-ui-react'

const CustomerInfo = ({customerInfo}) => {
  return (
    <Segment>
        <Item.Group>
            <Item>
                <Item.Content verticalAlign='bottom'>
                    <Header as='h1'>{customerInfo.billingFirstName} {customerInfo.billingLastName}</Header>
                    <br/>
                    <Header as='h3'>{customerInfo.email}</Header>
                    <br/>
                    <Header as='h3'>{customerInfo.billingState}</Header>
                </Item.Content>
            </Item>
        </Item.Group>
    </Segment>
  )
}

export default CustomerInfo
