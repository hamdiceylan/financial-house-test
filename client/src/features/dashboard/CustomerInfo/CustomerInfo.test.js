import React from 'react'
import { shallow } from 'enzyme'
import CustomerInfo from './CustomerInfo';


describe('CustomerInfo', () =>{
    const props = {
        customerInfo : {
            billingFirstName: 'first name',
            billingLastName: 'last name',
            email: 'email',
            billingState: 'state',
        }
    }
    const customerInfo = shallow(<CustomerInfo {...props} />);

    it('renders correctly', () => {
        expect(customerInfo).toMatchSnapshot();
    })

})

