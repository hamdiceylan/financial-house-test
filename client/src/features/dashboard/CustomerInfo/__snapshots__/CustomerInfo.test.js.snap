// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerInfo renders correctly 1`] = `ShallowWrapper {}`;
